### PR TITLE
fix: margin over milestone tabs and  tab value in select input

### LINKED ISSUE #3232: The UI of milestones needs some margin
## Description
There should be a bottom margin below the navigation section  on both screen ( mobile and web )

image

image

### Screenshots

_No response_

### Additional information

_No response_

### LINKED ISSUE #3234: Select navigation does not show current selected section
## How to reproduce this bug?
Open a Linkfree profile on web screen and select a different section other than `my links`
Then toggle to mobile screen,
you will notice that select will show `my links` but the list will be of other sections for example `milestones` and `events`

Here it is `milestones` section but select is showing `my links`

image

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by razaahmad333**
## Fixes Issue

Closes #3234 
Closes #3232 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### before
Screenshot_20230112_084630

### after
Screenshot_20230124_103748

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3313"></a>

### CODE REVIEW

**Comment by ManishBisht777** [comment]
can you attach a before and after screenshot as well

File: components/user/UserTabs.js [hunk 1]
```diff
@@ -49,28 +49,28 @@ export default function UserTabs({ tabs, setTabs, setUserData, userData }) {
           return aVal.toLowerCase() > bVal.toLowerCase()
             ? 1
             : aVal.toLowerCase() < bVal.toLowerCase()
-            ? -1
-            : 0;
+              ? -1
```
Comment (ManishBisht777) [inline components/user/UserTabs.js:52]: These are from a formatter(prettier), right?

**Review by ManishBisht777 — COMMENTED** [review]
Looks good to me :)

File: components/user/UserTabs.js [hunk 2]
```diff
@@ -92,7 +92,7 @@ export default function UserTabs({ tabs, setTabs, setUserData, userData }) {
           name="tabs"
           onChange={changeTab}
           className="block w-full rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500"
-          defaultValue={tabs.find((tab) => tab.current).name}
+          value={tabs.find((tab) => tab.current).name}
```
Comment (eddiejaoude) [inline components/user/UserTabs.js:95]: This change should not be part of this pull request

File: components/user/UserTabs.js (hunk 1 above)
Comment (eddiejaoude) [reply components/user/UserTabs.js:52]: I think these changes should not be in this pull request as it is a separate file with no changes related to this pull request

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Please see inline comments

File: components/user/UserTabs.js (hunk 1 above)
Comment (razaahmad333) [reply components/user/UserTabs.js:52]: > These are from a formatter(prettier), right?

yes

File: components/user/UserTabs.js (hunk 2 above)
Comment (razaahmad333) [reply components/user/UserTabs.js:95]: actually I resolved two issues in one pr 
because the changes were too little

Comment (eddiejaoude) [reply components/user/UserTabs.js:95]: little changes are good, otherwise it is more difficult for maintainers

**Comment by eddiejaoude** [comment]
Please add a screenshot for before and after

**Comment by razaahmad333** [comment]
> Please add a screenshot for before and after

sorry for the delay, I attached the screenshots

**Comment by eddiejaoude** [comment]
Thank you for your contribution.

There are too many changes in 1 pull request causing it to be blocked out of date.

Please make focused changes in the future and let's get them merged faster.

Please create a new PR(s) with changes
